OpenRA.Renderer.Sdl2/Sdl2GraphicsDevice.cs

```diff
@@ -241,6 +241,16 @@ public void SetBlendMode(BlendMode mode)
 					GL.BlendFuncSeparate(BlendingFactorSrc.DstColor, BlendingFactorDest.Zero, BlendingFactorSrc.One, BlendingFactorDest.OneMinusSrcAlpha);
 					ErrorHandler.CheckGlError();
 					break;
+				case BlendMode.AdditiveLessBright:
+					GL.Enable(EnableCap.Blend);
+					ErrorHandler.CheckGlError();
+					GL.BlendFunc(BlendingFactorSrc.OneMinusDstColor, BlendingFactorDest.DstAlpha);
```

Comment:
A quick google search suggests that `GL.BlendFunc(BlendingFactorSrc.OneMinusDstColor, BlendingFactorDest.One);` is occasionally defined as "Soft additive" blending.  How about changing this to that, and then naming it `BlendMode.SoftAdditive`?